Assets/World/Track/Scripts/RandomTrackGenerator.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+public class RandomTrackGenerator : TrackGeneratorCommon
+{
+    [SerializeField]
+    GameObject m_firstTrack; // Temporary, we will programatically generate the first track in the future.
+
+    protected override void GenerateTrack(int trackLength, float trackAltitude)
+    {
+        string[] availableTracks = new[]
+        {
+            "Track-Straight",
+            "Track-ShortRight",
+            "Track-SharpLeft"
+        };
+
+        var currentTrack = m_firstTrack;
```

Comment:
Needs to have consistently strongly typed variables throughout the entire code.
```suggestion
        TYPE currentTrack = m_firstTrack;
```